config/memory_persister.go

```diff
@@ -0,0 +1,49 @@
+// Licensed under the Apache License, Version 2.0
+// Details: https://raw.githubusercontent.com/maniksurtani/quotaservice/master/LICENSE
+
+package config
+
+import (
+	"errors"
+	"io"
+)
+
+type MemoryConfigPersister struct {
+	config  io.Reader
```

Comment:
Do we want a reference to the `Reader`, or do we want to read the bytes from the reader and hold a reference to the `[]byte`?